fedimint-client/src/lib.rs

```diff
@@ -1506,6 +1507,11 @@ impl Client {
         Ok(())
     }
 
+    pub fn recovery_progress(&self) -> impl Stream<Item = (ModuleInstanceId, RecoveryProgress)> {
```

Comment:
Docstring, and maybe rename to `get_recovery_progress_stream`?
Nouns <<< Verbs :D